active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+Breadcrumb control
+===
+
+# Background
+
+A "Breadcrumb" UX pattern provides an way to show a current position in context.
+For example in Windows File Explorer the current directory is shown with a breadcrumb,
+where clicking on any node takes you to that ancestor directory:
+
+![File Explorer breadcrumb example](images/file-explorer-breadcrumb.jpg)
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. 
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 the nodes have no option to open a drop-down menu, typically used to show the node's siblings.
+V2 will add this. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Represents a control that shows a list of items, for example the navigation trail to the current location.
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes.
+When there is not enough width to display the whole control,
+the leading nodes are replaced with a node that the user can click on,
+opening the missing nodes in a flyout menu.
+
+![Breadcrumb with nodes: Home, Documents, Design, Northwind, Images. The app is resized so that the Breadcrumb crumbles - then, clicking the ellipsis reveals the crumbled Home and Documents nodes](images/Breadcrumb_crumbling_flyout.gif)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Home", "Documents", "Design", "Northwind", "Images"}
+```
+
+If the last item in a Breadcrumb is text and is too long to fit it will be clipped
+(if the item type is a string it will be trimmed, meaning shortened and replaced with an ellipsis).
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+#### Responding to clicks
+
+This example shows an app with a Breadcrumb across the top, and a content area on the rest of the Window.
+The Breadcrumb shows a list of items. Clicking on an item navigates the content back, and updates
+the Breadcrumb.
+
+```xaml
+<Grid RowDefinitions="Auto,*">
+    <Breadcrumb ItemClicked="CrumbClicked" ItemsSource="{x:Bind Items, Mode=OneWay}" />
+    
+    <ContentControl Grid.Row="1" Content="{x:Bind CurrentItem, Mode=OneWay}">
+        ...
+    </ContentControl>
+</Grid>
+```
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    public ObservableCollection<Items> Items { get; }
```

Comment:
Type mismatch with the `NavigateToItems` below - this is a collection of `Items`, the Navigate accepts `Item`